README.md

```diff
@@ -1,27 +1,60 @@
 go-skeleton
 -----------
-A skeleton for go microservices
+This project is used as boilerplate example for the structure of a go microservice. A template to get you started so to speak.
+
+A skeleton for go microservices was the core idea, so that's where the name comes from.
 
 
 ## Motivation
 Besides avoiding to copy it from the last microservice I've written, a friend asked how I'd do it.
 That's is how I'm doing it right now
 
 
+## Download
+
+Clone the repository using 
+
+### SSH
```

Comment:
I believe the git part can be omitted. I believe it's ok to assume whoever is seen this repo know how to clone it

Reply:
Yes that's true, but it fills the section with life. Just putting the command there makes it look empty. And if you help even 1 person with the commands, that would already be enough.
But if you want, I'll remove the cloning section.

Reply:
I'd like to remove the cloning section. I understand your point, but I believe it creates more noise than value.

Reply:
:+1: